roles/django_app_docker/templates/env.j2

```diff
@@ -55,6 +55,10 @@ FLOWER_URL_PREFIX={{ django_app_docker_flower_url_prefix }}
 FLOWER_BASIC_AUTH={{ django_app_docker_flower_user }}:{{ django_app_docker_flower_password }}
 {% endif %}
 
+{% if django_app_docker_use_rabbitmq %}
+PUBLISH_BROKER_URL={{ django_app_docker_publish_broker_url }}
```

Comment:
not a standard envvar in our projects, so we can leave this out and instead override/specify it for the specific deploy

Reply:
removed it